kafka-zookeeper/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM alpine:3.12 AS kafka
+
+ENV SCALA_VERSION 2.12
+ENV KAFKA_VERSION 2.3.0
+
+WORKDIR /kafka
```

Comment:
Should we use `/opt/kafka` for consistency?

Reply:
done.